agent/src/main/java/kanela/agent/util/Manifests.java

```diff
@@ -29,6 +31,22 @@
 @Value
 public class Manifests {
 
+    public static <T> Option<String> getJarVersion(Class<T> clazz) {
+            String className = clazz.getSimpleName() + ".class";
```

Comment:
I think that we can use a `Try` construction instead of manually create the `Option` WDYT?

Reply:
yep, I underestimate vavr 😬